subseas_flex_fcst/maproom.py

```diff
@@ -166,7 +227,7 @@ def local_plots(n_clicks, click_lat_lng, latitude, longitude):
 
     # Forecast CDF
     fcst_q, fcst_mu = xr.broadcast(quantiles, fcst_mu)
-    fcst_dof = int(fcst_var.attrs["dof"])
+    fcst_dof = int(dofVar) #int(fcst_var.attrs["dof"])
```

Comment:
if you make dofVar int from the get-go, then you just need fcst_dof = dofVar. And you could use "fcst_dof" isntead of "dofVar" when calling read_cptdataset and you'd have nothing to do here.